src/web3client/liquidator.js

```diff
@@ -12,12 +12,14 @@ class Liquidator {
         return;
       }
       this.app.logger.debug(`running liquidation job`);
-      const checkDate = this.app.time.getTimeWithDelay(0);
+      const checkDate = this.app.time.getTimeWithDelay(this.app.config.ADDITIONAL_LIQUIDATION_DELAY);
```

Comment:
I don't understand this line. Is the delay now applied in a different way than before?

Reply:
Before was bind to token PIC selection, now makes sense to be a global again.